state/factory/workingset.go

```diff
@@ -663,12 +675,19 @@ func (ws *workingSet) pickAndRunActions(
 			if fCtx.UnfoldContainerBeforeValidate {
 				if container, ok := nextAction.Envelope.(action.TxContainer); ok {
 					if err := container.Unfold(nextAction, ctx, ws.checkContract); err != nil {
-						if caller := nextAction.SenderAddress(); caller != nil {
-							ap.DeleteAction(caller)
-						}
-						actionIterator.PopAccount()
+						removeCaller(ap, actionIterator, nextAction.SenderAddress())
 						continue
 					}
+					if nextAction.To() != nil {
+						var (
+							ioAddr, _ = address.FromBytes(nextAction.To().Bytes())
+							to        = ioAddr.String()
+						)
+						if to == address.StakingProtocolAddr || to == address.RewardingProtocol {
```

Comment:
how is this possible in pickAndRun?

Reply:
yeah, after discussion, this is not very meaningful, a malicious node can anyway modify his `pickAndRun` to send such a tx in container format, so removed